### ISSUE TITLE
You don't always want to UrlEncode your url segment substitutions

### ISSUE THREAD

**Issue by mattiasnordqvist** [mattiasnordqvist/Web-Anchor#59]
I am in a situation where I need to substitute a part of my url that is environment-dependant. I'm using a configuration file to supply a value for an url-segment-substitution handled by a parameterlisttransformer. Code like this

```csharp
[BaseLocation({location})]
public interface IApi
{
    [Get("resource")]
    public Task<HttpResponseMessage> Get()
}
```

in my test-environment, location is "test/api". In my production environment, location is "prod/api/v2"

I expected my urls to become test/api/resource and prod/api/v2/resource respectively, but the slashes were replaced by their url encoded equivalents %2F, which is not what I want in this case.

Because I have different number of slashes in my urls, I cannot replace {location} with multiple segments-substitutions, like {partA}/{partB}

**Comment by mattiasnordqvist** [comment]
When does it break?:
When you have an url-segment replacement going on and the value your putting there is containing slashes.
How to fix?
If the slashes are supposed to be a part of the url-path AS path-separators, then you should do nothing. Otherwise, if you really want them to be encoded to %2F you can use the ApiSettings and slip in a 'PreservePathInUrlSegmentParameters = false;'

**Comment by mattiasnordqvist** [comment]
Released in v2.2.60
